geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1592,6 +1596,15 @@ public Object selectValue(String queryPredicate) throws FunctionDomainException,
    */
   public void registerInterest(K key, InterestResultPolicy policy);
 
+
+  public default void registerInterestForAllKeys() {
+    registerInterestRegex(".*");
+  }
+
+  public default void registerInterestForKeys(Iterable<K> iterable) {
+    iterable.forEach(k -> registerInterest(k));
+  }
+
```

Comment:
Do we need to make similar changes for:

registerInterest(K key, boolean isDurable);
registerInterest(K key, boolean isDurable, boolean receiveValues);
registerInterest(K key, InterestResultPolicy policy, boolean isDurable, boolean receiveValues);
registerInterest(K key, InterestResultPolicy policy, boolean isDurable);